content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade-1-11.md

```diff
@@ -0,0 +1,297 @@
+---
+reviewers:
+- sig-cluster-lifecycle
+title: Upgrading kubeadm clusters from v1.10 to v1.11
+content_template: templates/task
+---
+
+{{% capture overview %}}
+
+This guide is for upgrading `kubeadm` clusters from version 1.10.x to 1.11.x and 1.11.x to 1.11.y where `y > x`.
+
+{{% /capture %}}
+
+{{% capture prerequisites %}}
+
+Before proceeding:
+
+- You need to have a functional `kubeadm` Kubernetes cluster running version 1.10.0 or higher in order to use the process described here. Swap also needs to be disabled. This cluster should use a self-hosted etcd and static control plane pods.
+- Make sure you read the [release notes](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG-1.11.md) carefully.
+- Note that `kubeadm upgrade` will not touch any of your workloads, only Kubernetes-internal components. As a best-practice you should back up what's important to you. For example, any app-level state, such as a database an app might depend on (like MySQL or MongoDB) must be backed up beforehand.
+
+{{< caution >}}
+**Caution:** All the containers will get restarted after the upgrade, due to container spec hash value being changed.
+{{< /caution >}}
+
+Also, note that only one minor version upgrade is supported. For example, you can only upgrade from 1.10 to 1.11, not from 1.9 to 1.11.
+
+{{< caution >}}
+**Caution:** The default DNS provider in 1.11 is [CoreDNS](https://coredns.io/) rather than [kube-dns](https://github.com/kubernetes/dns). 
+To keep `kube-dns`, pass `--feature-flags=CoreDNS=false` to `kubeadm upgrade apply`.
+{{< /caution >}}
+
+{{% /capture %}}
+
+{{% capture steps %}}
+
+## Upgrading your control plane
+
+Execute these commands on your master node (as root):
+
+1. 
+
+    ```shell
+    export VERSION=$(curl -sSL https://dl.k8s.io/release/stable.txt) # or manually specify a released Kubernetes version
+    export ARCH=amd64 # or: arm, arm64, ppc64le, s390x
+    curl -sSL https://dl.k8s.io/release/${VERSION}/bin/linux/${ARCH}/kubeadm > /usr/bin/kubeadm
+    chmod a+rx /usr/bin/kubeadm
+    ```
+
+    {{< caution >}}
+    **Caution:** Upgrading the `kubeadm` package on your system prior to upgrading the control plane causes a failed upgrade. 
+    Even though `kubeadm` ships in the Kubernetes repositories, it's important to install `kubeadm` manually. The kubeadm 
+    team is working on fixing this limitation. 
+    {{< /caution >}}
+
+
+    `kubeadm` is only needed on individual (non-master) nodes for joining the cluster.
+    It is not necessary to update kubeadm on nodes 
```

Comment:
I believe that @luxas mentioned that it is required for the worker nodes during a v1.10.x -> v1.11.x upgrade to migrate to the kubelet component config.

Reply:
Yes there will be a step to grab the config on the upgrade command that requires the latest version.